dags/property_definitions.py

```diff
@@ -300,50 +329,5 @@ def property_definitions_hourly_schedule(context):
     previous_hour = now.replace(minute=0, second=0, microsecond=0) - datetime.timedelta(hours=1)
     target_hour = previous_hour.isoformat()
 
-    return {
-        "ops": {
-            "ingest_event_properties": {"config": {"target_hour": target_hour}},
-            "ingest_person_properties": {"config": {"target_hour": target_hour}},
-        }
-    }
-
-
-# Add a config for running a backfill for a specific day
-def run_backfill_for_day(date_str: str):
-    """
-    Helper function to create a run config for a backfill for a specific day.
-
-    Args:
-        date_str: Date string in YYYY-MM-DD format
-
-    Returns:
-        Dagster run config for the backfill
-    """
-    start_date = f"{date_str} 00:00:00"
-    end_date = f"{date_str} 23:59:59"
-
-    return {
-        "ops": {
-            "ingest_event_properties": {"config": {"start_date": start_date, "end_date": end_date}},
-            "ingest_person_properties": {"config": {"start_date": start_date, "end_date": end_date}},
-        }
-    }
-
-
-# Helper to run backfill for a specific hour
-def run_backfill_for_hour(hour_str: str):
-    """
-    Helper function to create a run config for a backfill for a specific hour.
-
-    Args:
-        hour_str: ISO format datetime string for the hour (e.g., "2023-05-15T14:00:00+00:00")
-
-    Returns:
-        Dagster run config for the backfill
-    """
-    return {
-        "ops": {
-            "ingest_event_properties": {"config": {"target_hour": hour_str}},
-            "ingest_person_properties": {"config": {"target_hour": hour_str}},
-        }
-    }
```

Comment:
Removed these as the configuration is now a bit less repetitive and it's a bit easier to specify the time range as well